### ISSUE TITLE
Frontend Preview

### ISSUE THREAD

**Issue by Zeromax** [Zeromax/contao-sb-admin#17]
```
File system/themes/sb-admin/basic.css does not exist .. in system/modules/core/library/Contao/Combiner.php on line 134
```

see https://community.contao.org/de/showthread.php?59192-sb-admin-Alternatives-Backend-Theme&p=384677&viewfull=1#post384677

**Comment by Zeromax** [comment]
Bug is visible if `sb-admin` is not the global default Theme and only in use from the current User.

**Comment by Zeromax** [comment]
fixed in cc49eda
